Change from 18f to alphagov for modules

LeePorte opened the pull request:
Remove the 18f element from the modules. As this project is no longer being developed by 18f it makes sense to make our lives easier.

Commit:
Change from 18f to alphagov for modules

AP-Hunt reviewed (changes requested):
There's a path tweak in `.travis.yml` that needs attention for this to be complete

Commit:
Update travis config to remove install block

The complex install block is no longer required as we have changed the
module references
